.gitignore

```diff
@@ -8,19 +8,42 @@
 /Microsoft.Identity.Web/.vs
 /Microsoft.Identity.Web/bin
 /Microsoft.Identity.Web/obj
-/1. WebApp signs-in users with Azure AD (OIDC)/1.3. WebApp signs-in work and school or personal accounts/.vs
-/1. WebApp signs-in users with Azure AD (OIDC)/1.3. WebApp signs-in work and school or personal accounts/bin
-/1. WebApp signs-in users with Azure AD (OIDC)/1.3. WebApp signs-in work and school or personal accounts/obj
-/1. WebApp signs-in users with Microsoft Identity (OIDC)/1.1. in my org/.vs
-/1. WebApp signs-in users with Microsoft Identity (OIDC)/1.1. in my org/obj
-/1. WebApp signs-in users with Microsoft Identity (OIDC)/1.1. in my org/bin
-/2. WebApp calls Microsoft Graph on behalf of signed-in user/2.1. using authorization code flow/.vs
-/2. WebApp calls Microsoft Graph on behalf of signed-in user/2.1. using authorization code flow/obj
-/2. WebApp calls Microsoft Graph on behalf of signed-in user/2.1. using authorization code flow/bin
-/3. WebApp calls several APIS (incremental consent and CA)/.vs
-/3. WebApp calls several APIS (incremental consent and CA)/bin
-/3. WebApp calls several APIS (incremental consent and CA)/obj
-/1. WebApp signs-in users with Microsoft Identity (OIDC)/1.3. with work and school or personal accounts/.vs
-/1. WebApp signs-in users with Microsoft Identity (OIDC)/1.3. with work and school or personal accounts/obj
-/2. WebApp calls Microsoft Graph on behalf of signed-in user/2.2/.vs
-/2. WebApp calls Microsoft Graph on behalf of signed-in user/2.1. using authorization code flow/AppCreationScripts/Steps.md
+/1-WebApp-OIDC/1-1-MyOrg/.vs
+/1-WebApp-OIDC/1-1-MyOrg/bin
+/1-WebApp-OIDC/1-1-MyOrg/obj
+/1-WebApp-OIDC/1-2-AnyOrg/.vs
+/1-WebApp-OIDC/1-2-AnyOrg/bin
+/1-WebApp-OIDC/1-2-AnyOrg/obj
+/1-WebApp-OIDC/1-3-OrgsOrPersonal/.vs
+/1-WebApp-OIDC/1-3-OrgsOrPersonal/bin
+/1-WebApp-OIDC/1-3-OrgsOrPersonal/obj
+/1-WebApp-OIDC/1-4-Soveign/.vs
+/1-WebApp-OIDC/1-4-Soveign/bin
+/1-WebApp-OIDC/1-4-Soveign/obj
+/1-WebApp-OIDC/1-5-B2C/.vs
+/1-WebApp-OIDC/1-5-B2C/bin
+/1-WebApp-OIDC/1-5-B2C/obj
+/1-WebApp-OIDC/1-6-SignOut/.vs
+/1-WebApp-OIDC/1-6-SignOut/bin
+/1-WebApp-OIDC/1-6-SignOut/obj
+/2-WebApp-graph-user/2-1-Call-MSGraph/.vs
```

Comment:
We can omit "Call", right? What other purpose a sample will have with Graph other than calling it :)

Reply:
Let's keep it.